### DISCUSSION TITLE
Spanish Translation

### DISCUSSION THREAD

**Discussion by pocc** [bafolts/terraforming-mars#1346] [Localization]
This is per a discord request/someone volunteering.

**Comment by keros17** [comment]
Hello people!

I'm doing the translation from the fanmade gigaexpansion and after that I could help with the translation from the program to the spanish language.

If the developer wants also the fanmade content translated there is no problem to giving you that ^^

Best regards,
Gerard

**Comment by alrusdi** [comment]
@keros17 great! Ask me if you will need some help

**Comment by keros17** [comment]
Thanks @alrusdi ^^

**Comment by kberg** [comment]
This is done.
